### PR TITLE
feat(agora): migrate Nominated Targets (AG-1551)

### PR DESCRIPTION
**Pull request by sagely1**
## Description

- Adding Nominated Targets Component
** Note: table on page will be implemented in a future PR
- Removing teams dependency on legacy ApiService.  Moving to OpenApi implementation.
- Site-wide CSS updates (also using normalize.css to serve as a css reset)
- Added shared components - SvgIcon and ModalLink
   
## Testing
`nx test agora-nominated-targets`

## Expected output
Navigate to:
http://localhost:4200/genes/nominated-targets

image

### CODE REVIEW

File: apps/agora/api/.eslintrc.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     },
     {
       "files": ["*.js", "*.jsx"],
-      "rules": {}
+      "rules": {
+        "tailwindcss/no-custom-classname": "off"
```
Comment (sagely1) [inline apps/agora/api/.eslintrc.json:16]: Added rule as there was a linting error in webpack.config.js

Comment (tschaffter) [reply apps/agora/api/.eslintrc.json:16]: We could ignore `webpack.config.js` with:

```
"files": ["*.js", "*.jsx", "!webpack.config.js"],
```

Comment (tschaffter) [reply apps/agora/api/.eslintrc.json:16]: We should either ignore `webpack.config.js` or disable the rule just for `webpack.config.js`. The former is either. Note that `webpack.config.js` may be removed in the near future when we switch to esbuild.

Comment (sagely1) [reply apps/agora/api/.eslintrc.json:12]: sounds good!

**Review by tschaffter — APPROVED** [review]

**Pull request merged**